src/js/utils/strings.js

```diff
@@ -57,7 +57,7 @@ define([
             return azureFormat;
         }
 
-        path = path.substring(path.lastIndexOf('/') + 1, path.length).split('?')[0].split('#')[0];
```

Comment:
This change passes all the unit tests, but are there any URL's that will break because of this change?